src/app/services/api.ts

```diff
@@ -517,7 +612,12 @@ export class ApiService {
     return this.http.get<Object>(`${this.pathAPI}/${queryString}`, {});
   }
 
-  // NB: returns array with 1 element
+  /**
+   * Get comment period. Get a CP with a specified set of fields.
```

Comment:
```suggestion
   * Get comment period. Get a comment period with a specified set of fields.
```
Seems to be referred to this way everywhere else

Reply:
I'll revise the description here, it's a bit redundant too.